pages/index.js

```diff
@@ -36,12 +37,16 @@ function Home({ data, preview }) {
     return null
   }
 
-  const { sections, title } = pageData
+  const { sections, title, seo } = pageData
 
   return (
     <>
       <Head>
         <title>{title}</title>
+        <meta name="keywords" content={seo?.keywords} />
+        <meta name="synonyms" content={seo?.synonyms} />
+        <meta name="description" content={seo?.description} />
+        <meta name="image" content={urlFor(seo?.seoImage)} />
```

Comment:
Add other additional meta tags such as OG tags. Try using [metatags.io](https://metatags.io/) to get a better idea of what I'm talking about. 😊

Reply:
added open graph tags